### PR TITLE
[4.0] com_menus counts [a11y]

### PR DESCRIPTION
**Pull request by brianteeman**
This pr is similar to #34615 but this time for menus

To test apply PR and go to the list of menus (manage)

The count of menu items in the published, unpublished and trashed states now has a tooltip

image

cc @chmst

### CODE REVIEW

File: administrator/components/com_menus/tmpl/menus/default.php [hunk 1]
```diff
@@ -123,29 +123,62 @@
...
-										<span class="btn<?php echo ($item->count_published > 0) ? ' btn-success' : ' btn-secondary'; ?>">
-											<?php echo $item->count_published; ?></span>
+										<a class="btn<?php echo ($item->count_published > 0) ? ' btn-success' : ' btn-secondary'; ?>"
+											href="<?php echo Route::_('index.php?option=com_menus&view=items&menutype=' . $item->menutype . '&filter[published]=1'); ?>"
+											aria-describedby="tip-publish<?php echo $i; ?>">
+											<?php echo $item->count_published; ?>
+										</a>
+										<div role="tooltip" id="tip-publish<?php echo $i; ?>">
+											<?php echo Text::_('COM_MENUS_COUNT_PUBLISHED_ITEMS'); ?>
+										</div>
```
Comment (Quy) [inline administrator/components/com_menus/tmpl/menus/default.php]: These lines are same as lines 139-141. Lines 131-133 can be moved after line 142,  thus no need for lines 139-141.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on a4908013f5b68a38bf4250c0f631e5f33f7f0180<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34627">issues.joomla.org/tracker/joomla-cms/34627</a>.</sub>

**Comment by brianteeman** [comment]
@chmst should I do the same for content_maps and user groups?

**Comment by chmst** [comment]
@brianteeman yes, please
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34627">issues.joomla.org/tracker/joomla-cms/34627</a>.</sub>

**Comment by RickR2H** [comment]
I have tested this item :white_check_mark: successfully on a4908013f5b68a38bf4250c0f631e5f33f7f0180<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34627">issues.joomla.org/tracker/joomla-cms/34627</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34627">issues.joomla.org/tracker/joomla-cms/34627</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thank you

**Pull request merged**